classes/Translate.php

```diff
@@ -170,31 +173,36 @@ public static function getModuleTranslation($module, $originalString, $source, $
 
         $name = $module instanceof Module ? $module->name : $module;
 
-        $language = Context::getContext()->language;
+        if (!is_null($locale)) {
+            $iso = Language::getIsoByLocale($locale);
+        }
 
-        if (!isset($translationsMerged[$name]) && isset(Context::getContext()->language)) {
+        if (empty($iso)) {
+            $iso = Context::getContext()->language->iso_code;
+        }
+
+        if (!isset($translationsMerged[$name][$iso])) {
             $filesByPriority = array(
                 // Translations in theme
-                _PS_THEME_DIR_.'modules/'.$name.'/translations/'.$language->iso_code.'.php',
-                _PS_THEME_DIR_.'modules/'.$name.'/'.$language->iso_code.'.php',
+                _PS_THEME_DIR_.'modules/'.$name.'/translations/'.$iso.'.php',
+                _PS_THEME_DIR_.'modules/'.$name.'/'.$iso.'.php',
                 // PrestaShop 1.5 translations
-                _PS_MODULE_DIR_.$name.'/translations/'.$language->iso_code.'.php',
+                _PS_MODULE_DIR_.$name.'/translations/'.$iso.'.php',
                 // PrestaShop 1.4 translations
-                _PS_MODULE_DIR_.$name.'/'.$language->iso_code.'.php'
+                _PS_MODULE_DIR_.$name.'/'.$iso.'.php'
```

Comment:
These concatenation shouldn't be spaced?

Reply:
no norm for that, in this function is like that, so..